NewBank/newbank/server/NewBank.java

```diff
@@ -169,6 +172,9 @@ private String payRequest(CustomerID customerId, String[] splitRequest) {
         // 7. If true returned - add money to the second account
         toAccount.addMoney(amount);
 
+        // Add the current date to the list of successful payments
+        payments.add(LocalDate.now());
```

Comment:
Maybe it would be worth to store this information in the customer object, together with the information about the amount and whether its an incoming or outgoing transaction

Reply:
I think it's a great idea to store this information - it will be really useful to have this stored for the statement ticket next sprint if possible! :)